docker-compose.yml

```diff
@@ -11,6 +11,8 @@ services:
     environment:
       API_URL: http://web:3001
       ALLOWED_CORS_ORIGINS: "http://web:3001,http://localhost:3000"
+      NEXTGEN_ENTITIES_URL: "http://entities.czidnet:8008"
+      NEXTGEN_WORKFLOWS_URL: "http://workflows.czidnet:8042"
```

Comment:
We may not need to address this now, but are either of these services in deployed environments?  We'll eventually need to add the URLs in github env vars and configure the deploy actions to set those with `happy config`

Reply:
They aren't currently in dev/staging/prod but we're working on it. I'll make some noise when they're ready!